index.js

```diff
@@ -1,16 +1,10 @@
 module.exports = {
   extends: require.resolve('eslint-config-airbnb-base'),
   rules: {
-
     // Disallow comma-dangle
     // http://eslint.org/docs/rules/comma-dangle
     'comma-dangle': [2, 'never'],
 
-    // require the use of === and !==
-    // allow == null
-    // http://eslint.org/docs/rules/eqeqeq
-    eqeqeq: [2, 'allow-null'],
-
```

Comment:
This rule was made redundant. Upstream airbnb config has supported this since their v8.0.0 release, 2016-04-21.
https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/CHANGELOG.md#800--2016-04-21